firmware/application/apps/ui_looking_glass_app.hpp

```diff
@@ -93,8 +97,10 @@ namespace ui
             ChannelSpectrumFIFO* fifo { nullptr }; 
             uint8_t max_power = 0;
             int32_t steps = 250 ; // default of 250 Mhz steps
-            bool live_frequency_view = false ;
+            uint8_t live_frequency_view = 0 ;
```

Comment:
Why switch from book?

Reply:
because it's now having SPECTRUM (0) , LIVE-V (1) or PEAK-V (2) and since peak-v is a modified live-v I kept the var name and promoted it to an uint8_t

Reply:
Ah right, gotcha. Approved